libshvchainpack/include/shv/chainpack/accessgrant.h

```diff
@@ -55,5 +55,23 @@ struct SHVCHAINPACK_DECL_EXPORT UserLogin
 	chainpack::RpcValue toRpcValueMap() const;
 	static UserLogin fromRpcValue(const RpcValue &val);
 };
+
+struct SHVCHAINPACK_DECL_EXPORT AccessGrant
+{
+	using AccessLevel = MetaMethod::AccessLevel;
```

Comment:
Maybe AccessLevel should be refactored outside of MetaMethod? So that you don't have to use these aliases...

Reply:
I was thinking about this, but I cannot find good header to put it in. May be we can create `accesslevel.h`

Reply:
Yes, I agree. Headers are for free.